docs/pages/includes/database-access/guides-next-steps.mdx

```diff
@@ -1,4 +1,7 @@
 - Learn how to [restrict access](../../database-access/rbac.mdx) to certain users and databases.
+
 - View the [High Availability (HA)](../../database-access/guides/ha.mdx) guide.
+
 - Take a look at the YAML configuration [reference](../../database-access/reference/configuration.mdx).
+
 - See the full CLI [reference](../../database-access/reference/cli.mdx).
```

Comment:
Per the [commit message](https://github.com/gravitational/teleport/pull/17393/commits/938b5cf8fc625024d6271d431b1b1cef1f7e0647), I did this so additional ul items added will have consistent spacing.